stage0/src/main.rs

```diff
@@ -45,6 +45,18 @@ pub unsafe extern "C" fn SecureFault() {
 
 #[entry]
 fn main() -> ! {
+    cfg_if::cfg_if! {
+        if #[cfg(not(feature = "a0-hardware"))] {
```

Comment:
We probably want to do this assert in either case, changing the expected value -- because a 0A image won't work properly on 1B.

Also isn't it 0A instead of A0?